tests/unit/test_routing_fixes.py

```
@@ -0,0 +1,96 @@
+        self.assertIsNotNone(root_route_match, "Root route pattern not found in source code")
+        self.assertIsNotNone(api_init_match, "API initialization pattern not found in source code")
+
+        root_pos = root_route_match.start()
+        api_pos = api_init_match.start()
+        self.assertLess(root_pos, api_pos, "Root endpoint should be registered before Flask-RESTX initialization")
+
+    def test_test_files_fixed(self):
+        """Test that test files have been fixed with correct namespace definitions."""
+        # Test each file individually to avoid loops in tests
+        test_file = PROJECT_ROOT / 'deployment' / 'cloud-run' / 'test_swagger_debug.py'
+        if os.path.exists(test_file):
```

Ruff (0.12.2)

`os.path.exists()` should be replaced by `Path.exists()`

(PTH110)